<launch>

  <node pkg="localization" type="deadreckogning" name="deadreckogning">
    <param name="initial_x_pos" type="float" value="0.0"/>
    <param name="initial_y_pos" type="float" value="0.0"/>
    <param name="initial_z_angle" type="float" value="0.0"/>
  </node>

  <node pkg="localization" type="torviz" name="torviz">
    <param name="lidar_angle" type="float" value="0.0"/>
  </node>

  <include file="$(find rplidar_ros)/launch/view_rplidar.launch"/>

</launch>
